### DISCUSSION TITLE
Visualizing more than one rows of a Pandas DF

### DISCUSSION THREAD

**Discussion by gianpd** [elimintz/justpy#455] [Q&A] [answered]
Hi there! 

First of all, thanks for the project, it is awesome. 

I'm working on it since a couple of days, trying to upload and visualize a csv/parquet file. 

I've followed the pandas example, and looked to the documentation, in order to understand how to create an Ag Grid from a pandas dataframe. 

However, I am not able to visualize more than one row of  my DF. 

If I try to show more than one row (even just two rows)  I get an empty div in the web page. 

I've checked the grid.options which it seems to have all the field in a correct way. The rawdata are correctly populated. 

![image](https://user-images.githubusercontent.com/50718324/186399028-3d544c01-5267-49f1-bad5-6194a29f0eb8.png)

![image](https://user-images.githubusercontent.com/50718324/186398547-b88e3a98-c2f1-457e-8c62-ee1fb32dc514.png)

The first image is about my log, where the df.head() and the grid.options are printed. 
The second image reports the correct visualization if just one row is showed.

Thanks for the help!

**Comment by WolfgangFahl** [comment, marked as answer]
Please note that we are not using issues for questions/support like requests any more. Since justpy is an interface to javascript components we need to make sure first what the nature of the issues is and therefore it's suggested t to ask your question at https://stackoverflow.com/questions/tagged/justpy additionally adding the aggrid tag. Please follow the stackoverflow rules to supply a full explanation including source code. Later you might come back here with an issue that specifies what the correct justpy behavior should be.

**Comment by gianpd** [comment]
Ok, no problem. Anyway, for someone interested, I've founded that, actually, there is a limit of maximum number of rows/columns that could be showed with Ag-Grid. I solved, by limited these numbers.
